### PR TITLE
Restructure TYPO3_CONF_VARS

### PR DESCRIPTION
**Pull request by linawolf**
Use the full path in the confval, this removes the warnings about duplicate confvals and lists the
complete confvals in the index.

Beautify the menu

Leave hints on how and where to set them.

releases: main, 11.5

### CODE REVIEW

**Comment by brotkrueml** [comment]
I had that on my list since dev days ;-) Now, you are quicker :-)

**Review by brotkrueml — COMMENTED** [review]
At least for BE the configuration options are not ordered alphabetically. This means, that the TOC is mostly useless, as it cannot be used for a quick finding. So, a suggestion:

1. Restructure the pages without TOC and merge it also in v11.5.
2. Add a TOC and sort the options alphabetically only for main.

This way, we have less merge conflicts, and these can be handled easily (I assume).

**Comment by linawolf** [comment]
I don't see why the toc is useless just because it is not alphabetically ordered? After all it is still much faster to read the jump Menu then to scroll down and read each and every headline? The order is the same like in sysext/core/Configuration/DefaultConfigurationDescription.yaml So I  would hope there is a logic behind that order.

**Comment by linawolf** [comment]
The order when derived from the Localconfiguration.yaml is the same as is displayed in the Admin Tools:
image

**Comment by brotkrueml** [comment]
> The order when derived from the Localconfiguration.yaml is the same as is displayed in the Admin Tools: image

If there is a structure behind it, I don't get it.

It's okay, I don't have to agree with everything ;-)

**Comment by linawolf** [comment]
Yeah well me neither but I would suggest to change this in the Core and adjust the documentation accordingly

File: Documentation/Configuration/Typo3ConfVars/EXT.rst [hunk 1]
```diff
@@ -4,18 +4,38 @@
...
+
+The following configuration variables can be used to configure settings for
+the Extension manager:
+
+..  contents::
+    :local:
+
+..  attention::
+    Extension specific configuration should be stored in
+    :php:`$GLOBALS['TYPO3_CONF_VARS']['EXTENSION']` and not here.
```
Comment (DavidBruchmann) [inline Documentation/Configuration/Typo3ConfVars/EXT.rst:19]: I think this is not very clear, even I never know exactly what is meant by these words

File: Documentation/Configuration/Typo3ConfVars/GFX.rst [hunk 2]
```diff
@@ -5,20 +5,35 @@
...
 
-==================================
-$GLOBALS['TYPO3_CONF_VARS']['GFX']
-==================================
+============================
+GFX - graphics configuration
+============================
+
+The following configuration variables can be used to configure settings for
+the handling images and graphics:
```
Comment (DavidBruchmann) [inline Documentation/Configuration/Typo3ConfVars/GFX.rst:13]:
```suggestion
the handling of images and graphics:
```

**Review by DavidBruchmann — CHANGES_REQUESTED** [review]

File: Documentation/Configuration/Typo3ConfVars/EXT.rst (hunk 1 above)
Comment (linawolf) [reply Documentation/Configuration/Typo3ConfVars/EXT.rst:19]: Which words are you talking about?

**Review by DavidBruchmann — APPROVED** [review]

**Pull request merged**